### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by DeLaPF**

### CODE REVIEW

**Comment by DeLaPF** [comment]
Unsure how to make the program find the file (I created a txt file on my desktop and it wouldn't find it. The while loop will also not escape when I press enter and I'm unsure how to fix that as well. Everything else seems to be working properly.

**Comment by echew7** [comment]
lol nice. don't worry about file reading for now, you can just randomly generate a world to use. we can go over file reading when you get back, it's a bit annoying to handle with an IDE

File: Starter_Code/assign-1-life-updated/assign-1-life-updated/src/life.cpp [hunk 1]
```diff
@@ -31,14 +33,68 @@ static void welcome() {
     getLine("Hit [enter] to continue....   ");
 }
 
+static string promptConfigFile() {
+    string fileName = getLine("Please enter a config file [press enter if no Config]:");
+    while (!fileExists(fileName) || fileName != "") { //unsure why this does not work gets trapped in loop
```
Comment (echew7) [inline Starter_Code/assign-1-life-updated/assign-1-life-updated/src/life.cpp]: use a half while loop structure, it tends to be more clear:

```
while (true) {
    string fileName = getLine("Error: File does not exist. Please enter a config file [press enter if no Config]: ");
    if (fileExists(fileName) || fileName == "") {
        return fileName
    }
}
```

**Comment by DeLaPF** [comment]
Gotcha, what size should I make the world?

File: Starter_Code/assign-1-life-updated/assign-1-life-updated/src/life.cpp (hunk 1 above)
Comment (DeLaPF) [reply Starter_Code/assign-1-life-updated/assign-1-life-updated/src/life.cpp]: will do

**Comment by echew7** [comment]
Let's randomly generate it so the num rows/cols can be anything between 5 and 50.

**Comment by DeLaPF** [comment]
I used the code snippet provided, however it just wouldn't function at all.

**Pull request merged**